.github/workflows/python-package.yml

```diff
@@ -35,4 +35,4 @@ jobs:
         python -m pytest --cov=. --cov-report=xml
     - name: Analyze code with pylint
       run: |
-        pylint -rn -sn --recursive=y ./src
\ No newline at end of file
+        pylint -rn -sn --recursive=y --ignore=_version.py ./src
```

Comment:
This works just fine, clearly. I wonder if there is a cleaner way though - I'll spend some time checking if these GitHub actions will use settings from the pyproject.toml file. If so, we could probably put exclude files in there.